src/traces/scattermapbox/hover.js

```diff
@@ -34,6 +42,7 @@ module.exports = function hoverPoints(pointData, xval, yval) {
     function distFn(d) {
         var lonlat = d.lonlat;
         if(lonlat[0] === BADNUM) return Infinity;
+        if(trace.cluster.enabled && !clusteredPointsIds.includes(d.i + 1)) return Infinity;
```

Comment:
`includes` is not suported in IE.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes
You may use `indexOf` instead:
```js
if(trace.cluster.enabled && clusteredPointsIds.indexOf(d.i + 1) === -1) return Infinity;
```